controllers/clustergroupupgrade_controller.go

```diff
@@ -1043,11 +1043,12 @@ func (r *ClusterGroupUpgradeReconciler) validateCR(ctx context.Context, clusterG
 			Version: "v1",
 		})
 		err := r.Client.Get(ctx, client.ObjectKey{
-			Name: cluster,
+			Name:      cluster,
+			Namespace: cluster,
 		}, foundManagedCluster)
 
 		if err != nil {
-			return fmt.Errorf("Cluster %s is not a ManagedCluster", cluster)
+			return fmt.Errorf("Cluster %s is not a ManagedCluster!!!", cluster)
```

Comment:
Remove this.

Reply:
It seems you are adding exclamations, yet you self-commented to remove them? Is it 'reminder' you put yourself to address it shortly?